### PR TITLE
Zodiac CRRC Modifications

### PR DESCRIPTION
**Pull request by Zycorge**
Creates a stage for future boats/watercraft.
Creates a new class from the base NATO Assault Craft which is called MRB Zodiac which will show up under the 3d MRB Faction.

Increased MaxSpeed and EnginePower. This is an attempt to reflect zodiac speeds based on what I could determine from watching videos of them and using watercraft weight/horsepower calculations for the available specs for the CRRC. Those work out to suggest its top speed should be between 40 & 50 kph. After testing 50 feels way to quick. 

Max Speed in game is now ~40 - 45 kph which feels appropriately quick for the zodiac.

### CODE REVIEW

**Comment by Zycorge** [comment]
I'll also note that I tested this as a standalone mod with only a config.cpp file. Read through the way we have 3d_Assets laid out and then modified the code to fit that. Hopefully, its integrated correctly.

**Comment by fire-391** [comment]
Please place your new vehicle config in https://github.com/fire-391/3dMP/blob/master/3d_Assets/cfgVehicles/cfgGround.hpp 
and give it scope and faction params so it actually shows up in editor/Zeus.

**Comment by Zycorge** [comment]
I think I did that correctly?

File: 3d_Assets/cfgVehicles/cfgGround.hpp [hunk 1]
```diff
@@ -1195,4 +1195,17 @@
...
+class CfgVehicles {
+	class B_Boat_Transport_01_F;
+    class MRB_Zodiac: B_Boat_Transport_01_F {
+        displayName = "MRB Zodiac";
+		maxSpeed = 55;
+		enginePower = 47;
+		side = 1;
+		scope =2;
+		crew = "MRB_Element_CSO";
+		editorSubcategory = "B_3d_Marine_Raider_Vehicles";
```
Comment (fire-391) [inline 3d_Assets/cfgVehicles/cfgGround.hpp:1208]:
```suggestion
		editorSubcategory = "B_3d_Marine_Raider_Watercraft";

```

File: 3d_Assets/cfgVehicles/cfgGround.hpp [hunk 2]
```diff
@@ -1195,4 +1195,17 @@
...
+	};
+class CfgVehicles {
+	class B_Boat_Transport_01_F;
+    class MRB_Zodiac: B_Boat_Transport_01_F {
+        displayName = "MRB Zodiac";
+		maxSpeed = 55;
+		enginePower = 47;
+		side = 1;
+		scope =2;
+		crew = "MRB_Element_CSO";
```
Comment (fire-391) [inline 3d_Assets/cfgVehicles/cfgGround.hpp:1207]:
```suggestion
		crew = "MRB_Element_CSO";
                displayName = "MRB Zodiac";

```

**Review by fire-391 — CHANGES_REQUESTED** [review]

**Comment by fire-391** [comment]
Also need to add "MRB_Zodiac"; to https://github.com/fire-391/3dMP/blob/master/3d_Assets/config.cpp under units

**Review by fire-391 — APPROVED** [review]

**Pull request merged**